Move from Sinks to Inputs and Outputs (#65)
AMWA-TV/nmos-sink-metadata-processing: README.md

### What does it do?

- Introduces Inputs associated with Sender and Outputs associated with Receivers

> - Allows to configure media parameters of Senders through its Inputs basing on information about Outputs of Receivers

Reviewer: ```suggestion
- Allows the configuration of media parameters of Senders and Inputs using information coming from Receivers and Outputs.
```
